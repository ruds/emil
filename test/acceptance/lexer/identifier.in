abc+def
à
à
こんにちは世界
👋🏻
🫱🏽‍🫲🏿
hang ten🤙🏄dude11!1
$==?
'a'
foo_bar'
o'doyle
_foo
'
(*xx*)
Ꭳ
(*xx*)
birdieᎣ
(*xx*)
