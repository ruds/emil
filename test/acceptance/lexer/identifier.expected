0001 ID_WORD 616263 abc
0001 ID_OP 2B +
0001 ID_WORD 646566 def
0002 ID_WORD C3A0 à
0003 ID_WORD C3A0 à
0004 ID_WORD E38193E38293E381ABE381A1E381AFE4B896E7958C こんにちは世界
0005 ID_OP F09F918BF09F8FBB 👋🏻
0006 ID_OP F09FABB1F09F8FBDE2808DF09FABB2F09F8FBF 🫱🏽‍🫲🏿
0007 ID_WORD 68616E67 hang
0007 ID_WORD 74656E ten
0007 ID_OP F09FA499F09F8F84 🤙🏄
0007 ID_WORD 647564653131 dude11
0007 ID_OP 21 !
0007 ILITERAL 1 1
0008 ID_OP 243D3D3F $==?
0009 ERROR
0011 ID_WORD 626972646965 birdie
0011 ERROR
0013 END_OF_FILE
